### PR TITLE
add do_electrostatics switch

### PR DESCRIPTION
**Pull request by k9cdt**
Added a switch to turn off the real space electrostatics. This feature is needed in a theory we are proposing. The manuscript is now in preparation.

The code was first wrote by @alexmir18 in 2018-2020 upon an outdated branch of CP2K. It was maintained and merged into the current version by me.

### CODE REVIEW

**Comment by oschuett** [comment]
LGTM, but please expand `do_es` to `do_electrostatics` for improved readability.

File: src/fist_nonbond_force.F [hunk 1]
```diff
@@ -234,8 +234,8 @@ SUBROUTINE force_nonbond(fist_nonbond_env, ewald_env, particle_set, cell, &
                   fac_ei = 0.5_dp*fac_ei
                   fac_vdw = 0.5_dp*fac_vdw
                END IF
-               ! decide which interactions to compute
-               IF (do_multipoles) THEN
+               ! decide which interactions to compute\b
```
Comment (oschuett) [inline src/fist_nonbond_force.F:237]: typo?

Comment (k9cdt) [reply src/fist_nonbond_force.F:237]: Yes, sorry for that. Do you want me to correct it with another commit or you can overwrite it? Also, shall I add tests to it?

Comment (oschuett) [reply src/fist_nonbond_force.F:237]: Yes, just push another commit. I'll then squash them before merging. And tests are of course always welcome :-)

Comment (k9cdt) [reply src/fist_nonbond_force.F:237]: OK Got it!!

**Pull request merged**